keanu/models/items.py

```diff
@@ -0,0 +1,17 @@
+from keanu.app import flask_db as db
+from mongoalchemy.document import Index
+
+
+class Tags(db.Document):
```

Comment:
should we make a seperate table for this or just have a list of tags in the item table?